index.html

```diff
@@ -1,254 +1,234 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-	<meta charset="UTF-8">
-	<meta http-equiv="X-UA-Compatible" content="IE=edge">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Setup and mobile first</title>
-	<link rel="Stylesheet" href="Style.css">
-	<link rel="preconnect" href="https://fonts.googleapis.com">
-	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&family=Poppins:wght@400;500;600;700;800&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
-    <script src="https://kit.fontawesome.com/1a66ebfe2a.js" crossorigin="anonymous"></script>
-
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Version Destop day 2</title>
+  <link rel="stylesheet" href="style.css">
```

Comment:
- [ ] ⚠️ ⚠️ ⚠️ Please correct the path to your CSS file otherwise I can't review your project. Use **Style.css** instead of **s**tyle.css ⚠️ ⚠️ ⚠️ 

![image](https://user-images.githubusercontent.com/58642949/213592807-8a6ca853-89e6-4e2a-b729-dae2745d6953.png)